### DISCUSSION TITLE
Ollama environ value

### DISCUSSION THREAD

**Discussion by ByungKwanLee** [Holmeswww/AgentKit#27] [Q&A] [answered]
Hi I am the beginner to ollama and agentkit.

I came across os.environ.get("OLLAMA_URL") and os.environ.get("OLLAMA_TOKENIZER_PATH").

Could you provide what they would mean and what should be assigned?

Thanks in advance.

Oh, I got read the readme now.

![image](https://github.com/user-attachments/assets/7ff0adbc-aef5-4b50-add8-da00c0863415)

Until installing it, ollama in agentkit seem doesnt work.

**Comment by Holmeswww** [comment, marked as answer]
Hi,

Please note that agentkit does not automatically provide Ollama. Please install Ollama by following the README. Thanks for your interest in our work.
